scalafix/rules/src/main/scala/fix/Stable212Base.scala

```diff
@@ -66,11 +70,6 @@ trait Stable212Base extends CrossCompatibility { self: SemanticRule =>
         "scala.collection.Traversable" -> "scala.collection.Iterable"
       )
 
-    val linearSeqToList =
-      ctx.replaceSymbols(
-        "scala.collection.LinearSeq" -> "scala.collection.immutable.List",
-      )
```

Comment:
Why remove this rule?

Reply:
https://github.com/scala/scala-collection-compat/issues/99#issue-341891540